### DISCUSSION TITLE
How to use inline code for jupyter/html output

### DISCUSSION THREAD

**Discussion by jlperla** [quarto-dev/quarto-cli#8403] [Q&A] [answered]
### Description

I am trying to figure out how to create an exam template for jupyter output in qmd.  A simple example might be auto-incrementing question numbers, or adding up points for sub-questions.

I thought that the inline code might work, following https://quarto.org/docs/computations/inline-code.html but can't seem to get it to execute.  If I create the following file and then either do the HTML render or the `quarto test.qmd --to ipynb` it doesn't execute any of the inline code.  Python also didn't seem to work.

```qmd
      ---
      title: "Test"
      author: "Test"
      jupyter: julia-1.10
      
      ---
      
      ```{julia}
      #| echo: true
      point_counter = 0
      ```
      
      ## Question `{julia} point_counter`
      Points: `{julia} 100`
      
      Just making sure general execution works... it does
      ```{julia}
      #| echo: true
      4 + 4
      ```
```

Alternatively: if there are extensions/etc. to make exams and problem sets (with a jupyter-centric output) easier in qmd, I would love to see them.

**Comment by cscheid** [comment]
Are you on 1.4?

<img width="429" alt="image" src="https://github.com/quarto-dev/quarto-cli/assets/285675/bcafcfb2-7040-4cb4-8df5-82f63a8c8784">

**Reply by cscheid** [reply, marked as answer]
It's possible that you read about this _just now_, in which case we apologize for the slight documentation instability. The plan is to release 1.4 officially this week, and so there's a bit of a race condition at the moment.

**Comment by jlperla** [comment]
Ah sorry, still on 1.3.450 until the release.  I will retry when 1.4 is official.
